src/main/kotlin/org/jitsi/utils/Duration.kt

```diff
@@ -40,5 +44,59 @@ val Number.mins: Duration
 val Number.days: Duration
     get() = Duration.ofDays(this.toLong())
 
-operator fun Duration.times(x: Int): Duration = Duration.ofNanos(toNanos() * x)
+operator fun Duration.times(x: Number): Duration = this.multipliedBy(x.toLong())
```

Comment:
This accepts a `Number` but will not work as expected with e.g. `1.5`. I see that `kotlin.time.Duration` already has a `times(scale: Double)`. Can we use kotlin Duration everywhere?